src/tabbar.cpp

```diff
@@ -127,6 +129,32 @@ void TabBar::setNewTabButton()
     setTabButton(idx, QTabBar::RightSide, Q_NULLPTR);
 }
 
+int TabBar::realTabCount()
+{
+    // The last tab is "+" in TabBar, but that isn't a real tab which displays any content hence the real count is tab count - 1
+    if (count() - 1 < 0)
+        return 0;
```

Comment:
`count() - 1` runs the risk of underflowing if `count()` returns an unsigned type. `count() < 1` is a mathematically equivalent but computationally safer way of expressing the same idea.

Reply:
Eagle eye